doc/apiary/v2/fiware-ngsiv2-reference.apib

```diff
@@ -2181,6 +2181,7 @@ The payload may contain the following elements (all of them optional):
    JSON object with a `type` (a JSON string) and `value` (whose type depends on the `type`
    property).
    The different available scopes are described elsewhere.
++ `metadata`: a list of metadata names to include in the response. If omitted, it means "all metadata".
```

Comment:
Taking into account how already is described for metadata in subscriptions:

> metadata (optional): List of metadata to be included in notification messages. By default all metadata are included except the special ones described at "Special metadata in notifications" section. The string * can be used as member of this list to mean "all the custom (user) metadata".

I'd also suggest to include the text 

> The string * can be used as member of this list to mean "all the custom (user) metadata"

also in the case of this PR.